README.md

```diff
@@ -1,3 +1,42 @@
 # as
 
-MassaLabs monorepo for assemblyscript libraries.
+**MassaLabs** monorepo for assemblyscript libraries.
+_Massa AS_
+
+A collection of tools, objects and functions, specific in usage for Massa smart contracts, in *AssemblyScript.* 
```

Comment:
Nope. All things listed here can be used even if you're not developing SC on the Massa blockchain.